R/unidimensionalReliabilityFrequentist.R

```diff
@@ -1194,7 +1194,7 @@ gettextf <- function(fmt, ..., domain = NULL)  {
     cr[lower.tri(cr, diag = TRUE)] <- 0
     pos <- which(round(cr, 3) == 1, arr.ind = TRUE)
     if (length(pos) == 0) {
-      footnote <- gettextf("%s", footnote)
+      footnote <- footnote
```

Comment:
I think this line should just be removed.

```
if (length(pos) != 0) {
  for (i in seq_len(nrow(pos))) {
  ...

```

Reply:
yeah, I think so. `footnote <- footnote` is unnecessary.

Reply:
Bruno is correct. It works without the if statement. So you can just write 
```r
    cr <- cor(dataset, use = "pairwise.complete.obs")
    cr[lower.tri(cr, diag = TRUE)] <- 0
    pos <- which(round(cr, 3) == 1, arr.ind = TRUE)
    for (i in seq_len(nrow(pos))) {
      footnote <- gettextf("%sVariables %s and %s correlated perfectly. ",
                           footnote, variables[pos[i, 1]], variables[pos[i, 2]])
    }
```
Note that I also deleted the space before `Variables`

Reply:
yeah, but just small question: according to my understanding, there should be pairwise correlated perfectly for 'Loop printing', but in the view of the translator, I'm afraid may it be mistaken for three:`Variables %1$s, %2$s, and %3$s correlated perfectly`.If this can be ignored?

Do you have any good solutions? You can commit directly on this PR ;-)

Reply:
I have no idea what you mean :D. Writing `Variables %1$s, %2$s, and %3$s correlated perfectly` is also kind of strange to me (the numbers and $). And I have no head how the translator sees this. I just thought it would translate every string inside `gettext`, so `Variables` `and` `correlated perfectly` would be translated, wouldn't it? @boutinb?

Reply:
aha, no, I mean if the translator doesn't read the code, he may be mistaken for `Variables %1$s, %2$s, and %3$s correlated perfectly` (but it's not correct). I understand what `"%sVariables %s and %s correlated perfectly. "`means.

How about iterating this sentence `Variables %s and %s correlated perfectly.` to generate a list and added as footnote but just left `footnote` itself out of for loop?

Reply:
let's see a sample for it: 
Assuming that there are four variables, Q1-Q2(correlated perfectly) and Q3-Q4(correlated perfectly), there is no problem in printing according to the current code:

`Note.Variables Q1 and Q2 correlated perfectly.Variables Q3 and Q4 correlated perfectly.`

What I am afraid is that If the translator mistakenly thinks that the first placeholder is a variable, it will be interpreted as three variables and moved the position of first `%s` during translation. like `Variables %s, %s, and %s correlated perfectly`.

However, I believe this probability is very low, because our first placeholder `%s` is always in front.So I wrote it out to discuss if we could ignore the situation.

Reply:
I suppose we could also create a list and add it as a footnote. But does it hurt much if we err here? Say we merge this, can we then not check how the translator works before it gets into a new release? In any case, I hope @boutinb knows more.